rosbag2_transport/include/rosbag2_transport/player.hpp

```diff
@@ -339,6 +341,19 @@ class Player : public rclcpp::Node
   ROSBAG2_TRANSPORT_PUBLIC
   std::unordered_map<std::string, std::shared_ptr<rclcpp::GenericClient>> get_service_clients();
 
+  /// \brief Getter for clients corresponding to each action name
+  /// \return Hashtable representing action name to client
+  ROSBAG2_TRANSPORT_PUBLIC
+  std::unordered_map<std::string, std::shared_ptr<rclcpp_action::GenericClient>>
+  get_action_clients();
+
+  /// \brief Goal handle for action client is in processing.
+  /// \return true if goal handle is in processing, otherwise false.
+  /// \exception std::exception No action client is created for this action name.
+  ROSBAG2_TRANSPORT_PUBLIC
+  bool
+  goal_handle_in_process(std::string action_name, const rclcpp_action::GoalUUID & goal_id);
```

Comment:
I would like to ask to rename this function a  bit for the sake of clarity and better readability.
```suggestion
  ROSBAG2_TRANSPORT_PUBLIC
  bool
  is_goal_handle_in_processing(std::string action_name, const rclcpp_action::GoalUUID & goal_id);
```